src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -29,7 +29,7 @@ val RsFunction.isTest: Boolean
     get() {
         val stub = greenStub
         return stub?.isTest
-            ?: (queryAttributes.hasAtomAttribute("test") || queryAttributes.hasAtomAttribute("quickcheck"))
+            ?: (queryAttributes.hasMatchingAttribute(Regex("test")) || queryAttributes.hasAtomAttribute("quickcheck"))
```

Comment:
Let's extract `Regex("test")` into top-level property to avoid pattern compilation on each call